compiler/luci/pass/src/PropagateConcatenationQparam.test.cpp

```diff
@@ -237,3 +275,98 @@ TEST(PropagateConcatenationQparam, propagate_concat_quantparam_NEG)
   EXPECT_EQ(cg.input_1.at<loco::DataType::U8>(3), quantize(1, cg.input_1.quantparam()));
   EXPECT_EQ(cg.input_1.at<loco::DataType::U8>(4), quantize(2, cg.input_1.quantparam()));
 }
+
+TEST(PropagateConcatenationQparam, propagate_concat_quantparam_i16)
+{
+  // Check cases where qparam of concat_node is propagated
+  // (1) normal case: qparam is propagated to input_1 and input_2
+  // (2) input used by other Op: input_1 is an input of input_2. qparam is propagated only to
+  // input_2
+  // (3) subsequent concat: input_1 is concat. qparam is propagated only to input_2
+  // (4) const input: input_1 is const. constant values are quantized
+
+  // normal case: qparam of concat_node is propagated to input_1 and input_2
+  SimpleConcatGraph g(loco::DataType::S16);
+  luci::propagate_concat_quantparam(&g.concat_node, loco::DataType::S16);
+  EXPECT_FLOAT_EQ(g.concat_node.quantparam()->scale[0], 3.14);
+  EXPECT_EQ(g.concat_node.quantparam()->zerop[0], 0);
```

Comment:
@mhs4670go proposed #256 to use like 
```cxx
EXPECT_EQ(0, g.concat_node.quantparam()->zerop[0]);
```
Can you please revise these ?

Reply:
I didn't know the issue.. I revised them. Thanks.